internal/throttler/remote/throttler.go

```diff
@@ -34,13 +34,10 @@ const (
 )
 
 var (
-	errorUUIDNotSet = errors.New("Throttler uuid must be set")
+	errorClientIDNotSet = errors.New("Throttler ClientID must be set")
 )
 
-type creditResponse struct {
-	Operation string  `json:"operation"`
-	Credits   float64 `json:"credits"`
-}
+type creditResponse map[string]float64
```

Comment:
This isn't very future proof. If we add another field to the response (although I can't think of one at the moment), the json unmarshalling will fail. However, if we leave this as a struct, we can upgrade the agents to return the new field and still be backwards compatible with older versions of the client.

Reply:
Interesting. I was avoiding a backend change but maybe this is worth fixing. In that case, we might as well use Thrift to enforce structure here.

Reply:
Fix this one please